### PR TITLE
Some more pylab moves

### PR DESCRIPTION
**Pull request by dstansby**
First commit is moves, second commit is PEP8 cleans.

### CODE REVIEW

**Comment by dstansby** [comment]
Going to tag this as release critical since all the other pylab example moves are in the master branch and it would be good to get them all in one release. I think the rest can go in 1 more PR which I'll do in the next couple of days.

File: examples/images_contours_and_fields/custom_cmap.py [hunk 1]
```diff
@@ -72,7 +72,7 @@
 # Make some illustrative fake data:
 
 x = np.arange(0, np.pi, 0.1)
-y = np.arange(0, 2*np.pi, 0.1)
+y = np.arange(0, 2 * np.pi, 0.1)
```
Comment (choldgraf) [inline examples/images_contours_and_fields/custom_cmap.py]: the OCD person in me loves you for this haha

**Comment by choldgraf** [comment]
This looks great to me, will merge later today unless someone has other comments! Also @QuLogic for what it's worth Circle looks to be working beautifully!!!

**Comment by choldgraf** [comment]
merging!

**Pull request merged**